product_category_code_unique/data/product_category_sequence.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2021 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo noupdate="1">
+  <record id="seq_product_category_auto" model="ir.sequence">
+    <field name="name">Product Category</field>
+    <field name="code">product.category</field>
+    <field eval="4" name="padding" />
+    <field name="prefix">PC/</field>
```

Comment:
I guess a 
`        <field name="company_id" eval="False" />`
is needed.

Reply:
@rousseldenis ?

Reply:
Indeed. Done